---
controller_manager:
  ros__parameters:
    update_rate: 1000
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
      update_rate: 10
    effort_joint_controller:
      type: effort_controllers/JointGroupEffortController
effort_joint_controller:
  ros__parameters:
    joints:
      - front_sh
      - front_knee
      - front_ank
      - back_sh
      - back_knee
      - back_ank
    command_interfaces:
      - effort
    state_interfaces:
      - effort
      - velocity
      - position
    pid:
      p: 100.0
      i: 0.0
      d: 0.0
